### PR TITLE
Google Chrome Keyboard Shortcuts

### PR DESCRIPTION
**Pull request by NickCalabs**
---

IA Page: https://duck.co/ia/view/chrome_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@NickCalabs thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

**Comment by MrChrisW** [comment]
Hi @NickCalabs do you mind filling out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_goodie.md) and editing your initial comment? this will allow the community and staff to get a better idea of what your Instant Answer is all about. Thanks :+1:

File: lib/DDG/Goodie/ChromeCheatSheet.pm [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    'google chrome cheat sheet',
+    'help chrome',
+    'chrome reference',
+    'cheat sheet chrome',
+    'cheatsheet chrome',
+    'cheat sheet google chrome',
+    'cheatsheet google chrome',
+    'chrome shortcuts',
+    'google chrome shortcuts',
+    'goodle chrome keyboard shortcuts',
```
Comment (jagtalon) [inline lib/DDG/Goodie/ChromeCheatSheet.pm]: Typo: should be "google chrome ..."

**Comment by jagtalon** [comment]
@NickCalabs add a test like https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/t/EmacsCheatSheet.t

**Comment by jagtalon** [comment]
Broken on mobile:

screen shot 2015-04-10 at 9 55 56 am

**Comment by jagtalon** [comment]
@NickCalabs I think the cheatsheets is too complicated:

screen shot 2015-04-10 at 9 57 04 am

Can it be short and sweet like this?

screen shot 2015-04-10 at 9 56 35 am

**Comment by NickCalabs** [comment]
I'll get on the pull request template.
Perhaps I can just limit it to the more quick and simple shortcuts? I took it straight from Google's guide. There might be a shorter and more comprehensive list, I'll do some digging. If not, removing the superfluously long commands should be the answer?

**Comment by moollaza** [comment]
@NickCalabs just to let you know, we've now launched the simplified CheatSheets and some new documentation that explains how to create them.

All you need to do now is create a JSON file in /share/goodies/cheat_sheets/. You'll see other files there and they all follow the same syntax.

Instructions can be found here: https://duck.co/duckduckhack/goodie_cheat_sheets

---

Would you be interested in converting this to that format? If you could remove the other files and submit a JSON file in a new commit it would be awesome!

As well, I would suggest you try to be more succint with the the instructions and explanations. I don't think full sentences are quite necessary, I'm sure user's will be able to understand!

**Comment by AlfonzM** [comment]
Is anyone working on this? I would like to improve this and convert into the JSON format.

**Comment by javathunderman** [comment]
@AlfonzM I don't know, @NickCalabs are you interested in picking this up??

**Comment by NickCalabs** [comment]
Please do, I noticed the recent improvements with the JSON formatting, but
I won't get to it for a while.

**Comment by javathunderman** [comment]
@NickCalabs Could you close this then? (I don't have permissions).

**Comment by mintsoft** [comment]
@javathunderman I've closed it for now. @NickCalabs feel free to reopen it whenever you want!

**Pull request closed without merging**